.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
 public/components
-public/css
+css
```

Comment:
I moved the compiled css into the root of `public` as this is what it is in the library and therefore the image paths are set to that. However it should have been That should have been `public/*.css`